heimlig/src/common/jobs.rs

```diff
@@ -102,6 +112,10 @@ pub enum RequestType {
     SignExternalKey,
     Verify,
     VerifyExternalKey,
+    CalculateHmac,
```

Comment:
HMAC could be sorted right next to CMAC as they are both MACs. This affects the order of Requests/Responses below too.